plugins/file_selector/example/lib/open_image_page.dart

```diff
@@ -6,14 +6,15 @@ import 'package:flutter/material.dart';
 /// Screen that shows an example of openFiles
 class OpenImagePage extends StatelessWidget {
   void _openImageFile(BuildContext context) async {
-    final XTypeGroup typeGroup = XTypeGroup(
+    final typeGroup = XTypeGroup(
       label: 'images',
       extensions: ['jpg', 'png'],
     );

```

Comment:
Unnecessary for this PR, but looking at XTypeGroup definition, it could be made const, which would allow this to be made const.